### ISSUE TITLE
Investigate any changes needed to "Add Person" flow

### ISSUE THREAD

**Issue by NB28VT** [department-of-veterans-affairs/va.gov-team#62124]
## Overview

When checking the status of a user's data before starting the Form 526 flow, in cases where we haven't already, we submit an "add MVI" request to add the user to one of the other services the VA has for tracking veterans. This check happens before any of the "missing ID" messages we're addressing in this epic are displayed. However, depending on how that is implemented we need to make changes/consider how this fits into the new flow we are building in the epic

## Acceptance Criteria
- [ ] Determine if we need to make any changes to the Add Person flow
- [ ] Create a ticket to implement the changes if necessary

**Comment by NB28VT** [comment]
Potential contacts for the MVI flow mentioned in this ticket: https://app.zenhub.com/workspaces/526ez-bugs-and-defects-646e4436ff6787001ac4771a/issues/gh/department-of-veterans-affairs/va.gov-team/60331

**Comment by NB28VT** [comment]
@austinjprice this is the investigation we punted on [here](https://dsva.slack.com/archives/C05MPTGMX0E/p1692647266596259), so I think maybe this goes in one of the on hold columns?
